### PR TITLE
Badge: show avatars of pledgers

### PR DESCRIPTION
**Pull request by zegl**
### Summary

* Changes `Pledges` to be **public**. Some fields (refunded_at, scheduled_payout_at, hosted_invoice_url) are only shown to users with write access to the pledge.
* Added `pledge_badge_show_amount` to the public `Organization` object
* Show avatars of pledgers (max 4 shown) in the SVG badge

### CODE REVIEW

File: clients/packages/polarkit/src/components/badge/Badge.tsx [hunk 1]
```diff
@@ -211,6 +220,56 @@ export const Badge = ({
...
+                    backgroundColor: darkmode ? '#2e4070' : '#C9DBF4',
+                    color: darkmode ? '#a6c7ea' : '#4667CA',
+                    height: 22,
+                    width: 22,
+                    borderRadius: 22,
+                    marginLeft: '-6px',
+                    textAlign: 'center',
+                    fontSize: '8px',
+                    lineHeight: '20px',
+                    border: darkmode ? '1px solid #374e96' : '1px solid white',
```
Comment (birkjernstrom) [inline clients/packages/polarkit/src/components/badge/Badge.tsx]: Let's try to change the dark mode border to "#2e4070" same as the background. So it's more of a carve-out vs. border as is the case in the light mode.

**Comment by birkjernstrom** [comment]
Great work! Super excited to get this in-place (in hindsight, I can't believe we didn't have it from the get go).

Bit late so I focused the review now on the functionality & design vs. code implementation. I assume that's what was mainly requested too :-)

Just two things on that regard:
1. Let's change the border in dark mode to be the same gray as the background. Similar to how we're using white in the light version. So it looks more like a carve-out vs. actual border with the blue.
2. Dumb question, but for peace of mind: We're not exposing the email of the backer anywhere in this public API, right? Except potentially in other contexts if the request is from someone with write access?

In conversation today we discussed making the amount hidden. I've changed my mind. Since pledges are made public/social the amount is pseudo public by extension. Since an issue with one backer will automatically disclose the amount and for followers of it, it will be fairly easy to track the first 2-3 as well. So it's better that we have strict modes of public and anonymous later (if requested, e.g companies might be reasonable at certain stages/amounts). Instead of having subtle in-between nuances that will likely only serve to create confusion and mishaps.

**Comment by zegl** [comment]
> 2. Dumb question, but for peace of mind: We're not exposing the email of the backer anywhere in this public API, right? Except potentially in other contexts if the request is from someone with write access?

Nope, no emails are exposed. The `Pledger` object looks like this:

```python
class Pledger(Schema):
    name: str
    github_username: str | None
    avatar_url: str | None
```

File: clients/packages/polarkit/src/components/badge/Badge.tsx (hunk 1 above)
Comment (zegl) [reply clients/packages/polarkit/src/components/badge/Badge.tsx]: Nice idea, I guess you mean #3E3F43 in that case :)

**Pull request merged**